### ISSUE TITLE
Add breakpoint range media query mixin

### ISSUE THREAD

**Issue by mmjaeger** [twbs/bootstrap#18014]
I think there should be a mixin to allow for a range - there should be an easier way than this: @media (min-width: breakpoint-min(sm)) and (max-width: breakpoint-max(md)) {}

**Comment by cvrebert** [comment]
Well, there is
```scss
@include media-breakpoint-up($name, $breakpoints) {
  @include media-breakpoint-down($name, $breakpoints) {
    @content;
  }
}
```
But your suggestion seems reasonable IMO.

**Comment by cvrebert** [comment]
Fixed by #17317.
